### PR TITLE
Restore feature specs

### PR DESCRIPTION
**Pull request by Br3nda**
# What has changed and why?

# How to test this change

- [x] has automated tests
- [ ] at least one a reviewer ran the code

### CODE REVIEW

**Comment by Br3nda** [comment]
i had to reconfigure database cleaner. see a5712ed841619bd66079dfbdc2eb43b74701a696

File: spec/factories/rebate_forms.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@
     property { Property.find_by(valuation_id: valuation_id, rating_year: ENV['YEAR']) }
     # token <-- auto generated. Don't set in factory
     fields do
-      { full_name: 'Hermione Granger',
+      { full_name: Faker::Name.name,
```
Comment (CaraHill) [inline spec/factories/rebate_forms.rb]: 😭 😭 😭 
Poor Hermione.

Comment (Br3nda) [reply spec/factories/rebate_forms.rb]: Yeah, but they gotta have different names so we can tell we can't find her

**Comment by Br3nda** [comment]
These tests make calls to chrome, and they work on linux, but i haven't checked they work for macos devs
I've asked @CaraHill if she can run these on her mac, to check they work, before merging.

**Comment by Br3nda** [comment]
Cara reports it works on her mac

**Pull request merged**